src/octokit_utils/index.ts

```diff
@@ -1,127 +1,181 @@
-  const changedFiles: string[] = []
+async function getChangedFiles(
+  context: Context<Webhooks.WebhookPayloadPullRequest>
+) {
+  const changedFilesResponse = await context.github.pulls.listFiles(
+    initPullRelatedRequest(context)
+  );
+  const changedFiles: string[] = [];
   for (const changedFileData of changedFilesResponse.data) {
-    changedFiles.push(changedFileData.filename)
+    changedFiles.push(changedFileData.filename);
   }
-  context.log.info(`Changed files are: ${JSON.stringify(changedFiles)}`)
-  return changedFiles
+  context.log.info(`Changed files are: ${JSON.stringify(changedFiles)}`);
+  return changedFiles;
 }
 
-async function getPreviousReviewIds(context: Context<Webhooks.WebhookPayloadPullRequest>): Promise<ReviewLookupResult> {
-  const reviewsResponse = await context.github.pulls.listReviews(initPullRelatedRequest(context))
-  let hasReview: boolean = false
-  const reviewIds: Number[] = []
-  context.log.info(`Found ${reviewsResponse.data.length} reviews`)
+async function getPreviousReviewIds(
+  context: Context<Webhooks.WebhookPayloadPullRequest>
+): Promise<ReviewLookupResult> {
+  const reviewsResponse = await context.github.pulls.listReviews(
+    initPullRelatedRequest(context)
+  );
+  let hasReview: boolean = false;
+  const reviewIds: Number[] = [];
+  context.log.info(`Found ${reviewsResponse.data.length} reviews`);
   reviewsResponse.data.forEach((review) => {
-    context.log.info(review.user.login)
-    if (review.user.login === 'approveman[bot]' && review.state !== 'DISMISSED') {
-      hasReview = true
-      reviewIds.push(review.id)
+    context.log.info(review.user.login);
+    if (
+      review.user.login === "approveman[bot]" &&
+      review.state !== "DISMISSED"
+    ) {
+      hasReview = true;
+      reviewIds.push(review.id);
     }
-  })
-  return { hasReview, reviewIds }
+  });
+  return { hasReview, reviewIds };
 }
 
-async function dismissApproval(context: Context<Webhooks.WebhookPayloadPullRequest>, reviewId: Number): Promise<void> {
-  const req = initPullRelatedRequest(context)
-  req.review_id = reviewId
-  req.message = composeReviewDismissalMsg()
-  context.log.info('Try to dismiss the review')
-  const dismissResponse = await context.github.pulls.dismissReview(req)
-  context.log.info(`Dissmiss review #${reviewId} in PR #${req.pull_number} with status ${dismissResponse.status} and review state ${dismissResponse.data.state}`)
+async function dismissApproval(
+  context: Context<Webhooks.WebhookPayloadPullRequest>,
+  reviewId: Number
+): Promise<void> {
+  const req = initPullRelatedRequest(context);
+  req.review_id = reviewId;
+  req.message = composeReviewDismissalMsg();
+  context.log.info("Try to dismiss the review");
+  const dismissResponse = await context.github.pulls.dismissReview(req);
+  context.log.info(
+    `Dissmiss review #${reviewId} in PR #${req.pull_number} with status ${dismissResponse.status} and review state ${dismissResponse.data.state}`
```

Comment:
Codacy found an issue: [Exceeds maximum line length of 120](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6083797)